contrib/packaging/deb/changelog_parser.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import sys
+from docutils.core import publish_doctree
+from datetime import datetime
+
+
+class ParseSection(object):
+
+    @property
+    def date(self):
+        return datetime.strptime(self._meta.get("date"), "%Y-%m-%d")
+
+    @property
+    def unixtime(self):
+        return int(self.date.strftime("%s"))
+
+    def __init__(self, section):
+        self._section = section
+
+    def parse(self):
+        if self._section.tagname != "section":
+            return False
+
+        children = self._section.children
+
+        self._parse_title(children[0])
+        self._parse_metadata(children[1])
+        self._parse_changelog(children[2:])
+        return self
+
+    def _parse_title(self, title):
+        self._title = title.astext()
```

Comment:
Could extract the version here instead of using metadata?

Reply:
Yep, but keeping the other metadata? or you don't want to have a metadata in the news file?